src/components/axes/timeAxis.ts

```diff
@@ -207,6 +202,19 @@ export module Axis {
         return this._possibleTimeAxisConfigurations;
       }
       this._possibleTimeAxisConfigurations = configurations;
+      this._numTiers = _Util.Methods.max(this._possibleTimeAxisConfigurations.map((config: TimeAxisConfiguration) => config.length), 0);
+
+      if (this._isAnchored) {
+        this._createDomElements();
+      }
```

Comment:
Instead of this, is there a way we can just route the setter to go back to the _setup function rather than creating them on the fly?

It's just more conventional for us to do the minimum to affect state in our setters and then calling the right function that will handle the rest in terms of DOM manipulation